src/pkg/leaderElection.go

```diff
@@ -43,8 +44,34 @@ func RunLeaderElection(client *clientset.Clientset, lockName, lockIdentity, lock
 				log.Info().Msgf("Perform Migration")
 				for {
 					log.Info().Msgf("leader is %s", lockIdentity)
-					log.Info().Msgf("Getting replica count...")
-					time.Sleep(5 * time.Second)
+					replicaCount, err := getReplicas()
+					if err != nil {
+						log.Info().Msgf("Failed to get replica count: %v", err)
+						continue
+					}
+					log.Info().Msgf("Setting replica count to %v", replicaCount)
```

Comment:
```suggestion
					log.Info().Msgf("Ideal replica count should be %v", replicaCount)
```

Since this is prior to actually setting the replica count - its more speaking about what the "ideal" count should be set to as returned by the opslevel api.